forge/routes/auth/oauth.js

```diff
@@ -414,7 +414,18 @@ module.exports = async function (app) {
         }
     }, async (request, reply) => {
         if (request.params.ownerType === request.session.ownerType && request.params.ownerId === request.session.ownerId) {
-            reply.code(200).send()
+            let response
+            if (request.headers['ff-quota']) {
+                const project = await app.db.models.Project.byId(request.session.ownerId)
+                const team = await app.db.models.Team.byId(project.Team.id)
+                const fileStorageLimit = team.TeamType.getFeatureProperty('fileStorageLimit', 100)
+                const contextLimit = team.TeamType.getFeatureProperty('contextLimit', 1)
```

Comment:
```suggestion
                const project = await app.db.models.Project.byId(request.session.ownerId)
                const teamType = await project.Team.getTeamType()
                const fileStorageLimit = teamType.getFeatureProperty('fileStorageLimit', 100)
                const contextLimit = teamType.getFeatureProperty('contextLimit', 1)
```

Aware this is potentially a hot-path, this will slightly reduce the database work